flexmeasures_openweathermap/utils/locating.py

```diff
@@ -100,11 +99,11 @@ def find_weather_sensor_by_location(
             location[1] - weather_station.location[1]
             > max_degree_difference_for_nearest_weather_sensor
         ):
-            current_app.logger.warning(
+            raise Warning(
```

Comment:
I believe [this stops the program](https://stackoverflow.com/a/3891852), or not? Why are you choosing to `raise` here?

Reply:
I just wanted to stop the program which works on the tests. But this is the point of conflict between  tests and main logic where it gnores the missing sensors during saving forecasts. I will revert it back.